numba_dpex/core/targets/dpjit_target.py

```diff
@@ -34,31 +34,27 @@ def __init__(self, typingctx, target=DPEX_TARGET_NAME):
 
     @global_compiler_lock
     def init(self):
-        self.is32bit = utils.MACHINE_BITS == 32
-        self._internal_codegen = JITCPUCodegen("numba.exec")
         self.lower_extensions = {}
+        super().init()
+
         # TODO: initialize nrt once switched to nrt from drt. Most likely we
         # call it somewhere. Double check.
         # https://github.com/IntelPython/numba-dpex/issues/1175
         # Initialize NRT runtime
         # rtsys.initialize(self) # noqa: E800
-        self.refresh()
+        # self.refresh() # noqa: E800
```

Comment:
If the `refresh` call is not needed then why not remove instead of comment out?

Reply:
I was unsure if it is needed for nrt. If not - I can just remove it.

Reply:
Its purpose was to load any `dpjit`-specific function registries when the target is initialized. I see that the `refresh` function is removed as well. Let us remove, we will revisit when we have `dpjit`-specific registries.